src/models/reposyncstate.ts

```diff
@@ -51,7 +51,9 @@ export class RepoSyncState extends Model {
 			where: {
 				pullStatus: "complete",
 				branchStatus: "complete",
-				commitStatus: "complete"
+				commitStatus: "complete",
+				buildStatus: "complete",
+				deploymentStatus: "complete"
```

Comment:
@mboudreau 
> @joshkay10 don't forget to change the `countSyncedReposFromSubscription` in RepoSyncState with your new statuses :)

This should do the trick?

Reply:
yup yup :D